docker/nginx/conf.d/server.local.conf

```diff
@@ -0,0 +1,14 @@
+server {
+    # local server - no not expose this port externally
+    listen 8000;
+    listen [::]:8000;
+
+    # secure traffic by limiting to only local networks
+    allow 10.0.0.0/8;
+    allow 127.0.0.1/32;
```

Comment:
Shouldn't this only be 127.0.0.1/32 and 10.0.0.0/8? 

We might want to keep this to only the essential ones to prevent other servers in the datacenter from stumbling across this.

Reply:
I think even this would do

```
allow 127.0.0.1/32;  # localhost
allow 10.10.10.0/24; # docker network
deny all;
```